text/0040-pagination-convention.md

```diff
@@ -0,0 +1,312 @@
+                "name": "Settings",
+                "settings": {
+                    "ranking_rules": "Nothing",
+                    "distinct_attribute": "Nothing",
+                    "primary_key": "Nothing",
+                    "searchable_attributes": {
+                        "Update": [
+                            "citation"
+                        ]
+                    },
+                    "displayed_attributes": "Nothing",
+                    "stop_words": "Nothing",
+                    "synonyms": "Nothing",
+                    "attributes_for_faceting": "Nothing"
+                }
+            },
+            "duration": 0.238216885,
+            "enqueuedAt": "2021-05-07T11:25:43.073716593Z",
+            "processedAt": "2021-05-07T11:25:43.325663019Z"
+        },
+        {
+            "status": "processed",
+            "updateId": 3,
+            "type": {
+                "name": "ClearAll"
+            },
+            "duration": 0.001705567,
+            "enqueuedAt": "2021-05-13T09:59:14.406552884Z",
+            "processedAt": "2021-05-13T09:59:14.420332760Z"
+        }
+    ],
+    "offset": 2,
+    "limit": 2,
+    "total": 198,
+    "size": 2
+]
+```
+
+#### Get last page of processed updates
+
+##### Request
+
+`/indexes/:index_uid/updates?status=processed&offset=50&limit=10`
+
+> For the example, let's imagine the total number of processed updates is `60` and we want page of `10` items.
+
+##### Response
+```
+[
+    "data": [
+        {
+            "status": "processed",
+            "updateId": 49,
+            "type": {
+                "name": "Settings",
+                "settings": {
+                    "ranking_rules": "Nothing",
+                    "distinct_attribute": "Nothing",
+                    "primary_key": "Nothing",
+                    "searchable_attributes": {
+                        "Update": [
+                            "citation"
+                        ]
+                    },
+                    "displayed_attributes": "Nothing",
+                    "stop_words": "Nothing",
+                    "synonyms": "Nothing",
+                    "attributes_for_faceting": "Nothing"
+                }
+            },
+            "duration": 0.238216885,
+            "enqueuedAt": "2021-05-07T11:25:43.073716593Z",
+            "processedAt": "2021-05-07T11:25:43.325663019Z"
+        },
+        ...
+        {
+            "status": "processed",
+            "updateId": 59,
+            "type": {
+                "name": "ClearAll"
+            },
+            "duration": 0.001705567,
+            "enqueuedAt": "2021-05-13T09:59:14.406552884Z",
+            "processedAt": "2021-05-13T09:59:14.420332760Z"
+        }
+    ],
+    "offset": 50,
+    "limit": 10,
+    "total": 60,
+    "size": 10
+]
+```
+
+#### Requirements
+
+- ✅ Each paginable route must return the `total` number of items that match the query and the number of items returned in the response as `size`.
```

Comment:
can we consider using `count` instead of `size`?

Reply:
I hesitated between `count` and `size`. After thinking about it I thought that `count` could be misunderstood with `total`. In the context of a page, I thought that page size makes more sense than page count.